### ISSUE TITLE
[BUG]: possible false positive - combobox has the 'aria-autocomplete' attribute incorrectly set

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1095]
### Project

a11y checker engine

### Browser

Chrome

### Operating system

MacOS

### Description

there are two violations coming from the FilterableMultiSelect Carbon component
1. combobox has the 'aria-autocomplete' attribute incorrectly set on an element within the popup referenced by "<x>"  
2. popup of the combobox has DOM focus or has 'aria-activedescendant' defined, which is not allowed

It is re-producible on the storybook links [v10](https://v7-react.carbondesignsystem.com/?path=/story/components-multiselect--filterable) and [v11](https://react.carbondesignsystem.com/?path=/story/components-multiselect--filterable-with-layer).

Are there any suggestions to fix these or is it a carbon a11y issue?

**Rule ID**: **_combobox_autocomplete_**
**Reason ID**: **_Fail_1_**

### Steps to reproduce

Steps to re-produce on the storybook:
1. Pre-req seems to be when there are 2 or more multiselect filterable components on a page
2. Input some text (e.g. abc) on the first multiselect box and keep the cursor/focus on that same box
3. Run the IBM a11y scan, and you should be able to see the two critical violations I mentioned

![image](https://user-images.githubusercontent.com/16193609/193864161-b6c51892-8cd1-43d2-864a-e07dbd6bfa2f.png)

**Comment by philljenkins** [comment]
See Slack thread by @Kenneth Nguyen https://ibm-studios.slack.com/archives/C036P1CTN/p1664816141204579

It looks like the violation doesn't appear if there is one/more options from the dropdown when running the scan while the box is focused
![image](https://user-images.githubusercontent.com/16193609/193865937-0e3d9afa-8f82-4cd2-be9a-4de61f1d0508.png)

**Comment by philljenkins** [comment]
Element triggering issue:

`<input 
class="cds--text-input" 
role="combobox" 
aria-expanded="true" 
aria-haspopup="listbox" 
aria-autocomplete="list" 
autocomplete="off" 
id="carbon-multiselect-example-3-input" 
aria-describedby="filterablemultiselect-helper-text-id-1" 
value="abc" 
aria-owns="carbon-multiselect-example-3__menu" 
aria-controls="carbon-multiselect-example-3__menu">`

**Comment by philljenkins** [comment]
@shunguoy also noticed that we may need to add a test case for this situation to be checked.
